index.bs

```diff
@@ -230,15 +268,41 @@ Report Element Timing {#sec-report-element}
     1. <a>Queue the PerformanceEntry</a> |entry|.
 </div>
 
+Report text Element Timing {#sec-report-text}
+--------------------------------------------------------
+
+<div algorithm="report text element timing">
+    When asked to <dfn>report text element timing</dfn> given an {{Element}} |element| and a timestamp |renderTime|, perforrm the following steps:
```

Comment:
I think you have too many Rs in "perforrm" (or maybe not enough, as it can have a good ring to it :D)

Reply:
:D fixed